staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config_test.go

```diff
@@ -1871,36 +1822,6 @@ func Test_kmsv2PluginProbe_rotateDEKOnKeyIDChange(t *testing.T) {
 		wantLogs         []string
 		wantErr          string
 	}{
-		{
-			name:        "happy path, no previous state",
-			service:     &testKMSv2EnvelopeService{keyID: "1"},
-			state:       envelopekmsv2.State{},
-			statusKeyID: "1",
-			wantState: envelopekmsv2.State{
-				EncryptedObject:     kmstypes.EncryptedObject{KeyID: "1"},
-				ExpirationTimestamp: now.Add(3 * time.Minute),
-			},
-			wantEncryptCalls: 1,
-			wantLogs: []string{
-				`"encrypting content using envelope service" uid="panda"`,
-				fmt.Sprintf(`"successfully rotated DEK" uid="panda" useSeed=false newKeyIDHash="sha256:6b86b273ff34fce19d6b804eff5a3f5747ada4eaa22f1d49c01e52ddb7875b4b" oldKeyIDHash="" expirationTimestamp="%s"`,
```

Comment:
You need to come up with a way to not delete these tests.  We cannot lose coverage for the `useSeed=false` path.